js/change-event-list.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+(()=>{
+
+  const deleteEvent = (eventId, eventName) => { 
+    eventList.forEach((item) => {
+      item.id === eventId && clearTimeout(item.action) || clearInterval(item.action); 
+    });
+    eventList = eventList.filter((item) => {
+      return (item.name !== eventName && item.id !== eventId)
+    }); 
+  }
+
+  const renameEvent = (id, eventName, newEventName) => {
+    eventList.map((item) => { 
+      if(item.id === id && item.name === eventName){
+        return item.name = newEventName
+      }
+    });
+  }
+
+  const changeEventDate = (eventId, eventName, newEventTime) => {
+    let callback;
+    eventList.forEach((item) => {
+      item.id === eventId ? callback = item.function : "there is no this event"; 
```

Comment:
incorrect use of the ternary operator. You can use:
`const callback = item.id === eventId ? item.function : "there is no this event";`